core/templates/dev/head/forms/formBuilder.js

```diff
@@ -723,6 +723,7 @@ oppia.config(['$provide', function($provide) {
       var _openCustomizationModal = function(
           customizationArgSpecs, attrsCustomizationArgsDict, onSubmitCallback,
           onDismissCallback, refocusFn) {
+        document.execCommand('enableObjectResizing', false, false);
```

Comment:
(FYI, note that [https://docs.angularjs.org/api/ng/service/$document]($document) is a standard angular dependency.)